docs/frontend/understanding-the-style-directory.md

```diff
@@ -0,0 +1,101 @@
+    - `form/` - contains components related to form elements, wrappers and structures (form line, error messages,...)
+    - `in/` - contains inline components (product flag, messages, paging,...). Here belong all components that are not related to complex of information.
+    - `list/` - contains components related to list that are not created by HTML element table (product list, menu, categories,...). Here belong all components that are a list of items and are not created by HTML element table.
+    - `table/` - contains components related to table HTML element (product parameters, cart,...). Here belong all components that are a list of items and created by HTML element table.
+    - `window/` - contains components for showing window (popup)
+    - `wrap/` - contains components that wrap elements
+- `core/` - contains core setting
+    - `form/` - contains styles related to form elements
+        - `btn.less` - definition of `.btn` class and its modifications
+        - `input.less` - definition of base input HTLM element, `.input` class and its modifications
+    - `mixin/` - contains usefull `Less mixins`
+    - `animation.less` - used for CSS animation
+    - `base.less` - set base styles for HTML elements
+    - `reset.less` - reset CSS attributes of HTML elements
+    - `typography.less` - set base styles for typography
+    - `variables.less` - contains variables definition
+- `helpers/` - contains helper classes for setting `display`, `font-size`, `cursor`, `float`, `text-align`, `clear`, `margin`, `padding` attributes
+- `layout/` - contains styles related to web layout
+    - `header/` - contains all classes related to `.header` class
+- `libs/` - contains styles of third side application
+- `print/` - contains styles intended for print page. Structure of folder `print/` copy folder structure of this root folder. You will find in `print/` only files that were needed to be modified in order to show print page correctly.
+
+## How to create new LESS component
+First of all, is good to select a class name that will fit the purpose of the `LESS component`.
+
+After that you have to realize which type of component it should be. To determine where a new component belongs, it is recommended to follow the suggestion mentioned in the previous section **Root directory**. You can find these suggestions in the description for individual component folders.
+
+## How to style print page
+For this purpose please access `print/` folder. In the folder place only files that are necessary to style print page. It is recommended to respect folder structure as is in the root folder.
+
+### Best practices
+In case you want to hide HTML element there exist CSS class `.dont-print`. It is best practice to do not hide element through styles but by placing this class to an appropriate element.
```

Comment:
```suggestion
In case you want to hide HTML element there exist CSS class `.dont-print`. It is best practice not to hide element through styles but to place this class to an appropriate element.
```